src/uct/cuda/cuda_ipc/cuda_ipc_md.h

```diff
@@ -52,4 +63,11 @@ typedef struct uct_cuda_ipc_key {
         }                                                               \
     } while(0);
 
+#define UCT_CUDA_IPC_DEVICE_GET_COUNT(_num_device)                        \
+    do {                                                                  \
+        if (UCS_OK != UCT_CUDADRV_FUNC(cuDeviceGetCount(&_num_device))) { \
```

Comment:
can it return `CUDA_ERROR_NOT_READY`  (which would be translated to `UCS_INPROGESS`)?

Reply:
cuDeviceGetCount will not return CUDA_ERROR_NOT_READY